components/scream/cmake/Kokkos.cmake

```diff
@@ -8,8 +8,10 @@ else ()
   set(KOKKOS_SRC    ${CMAKE_SOURCE_DIR}/../../externals/kokkos)
   set(KOKKOS_BINARY ${CMAKE_BINARY_DIR}/kokkos/build)
 
-  # We want Kokkos to be in debug mode if scream is in debug mode
-  if (CMAKE_BUILD_TYPE_ci STREQUAL "debug")
+  # We want Kokkos to be in debug mode if scream is in debug mode. This is a bit hacky
+  # since we can't use KOKKOS_GMAKE_DEVICES yet.
+  set(NO_DEBUG_ARGS Volta70 Pascal60)
```

Comment:
Was wondering if anyone knew a better way to do this?

Reply:
I think this is a fairly reasonable way of doing this. An alternative would be to check the SCREAM_DEBUG variable, which is available in cmake already (but one needs to enable it explicitly).

On a side note: Why don't we want to enable kokkos debug for GPU builds?

Reply:
Kokkos debug build limits the number of threads per kernel. Since Kokkos debug is on in other builds to catch out of bounds access, for the GPU build we decided to leave debug off so kernels can run with maximum number of threads. Other approaches are reasonable, though.

Reply:
Ah right, I forgot about that.